examples/randomized-benchmark/DRAG-optimization/configuration.py

```diff
@@ -1,40 +1,72 @@
 import numpy as np
 
-gauss_len = 100
+gauss_len = 16
```

Comment:
why 16? because it is being scanned in the file?

Reply:
I put a short one because usually DRAG pulses are used when the length is small, not really in the range of 100 ns